scss/_utilities/_variables.scss

```diff
@@ -87,7 +87,7 @@ $font-large: type-scale(2); // 28.125px
 $font-medium: type-scale(1); // 22.500px
 $font-regular: type-scale(0); // 18.000px
 $font-small: type-scale(-1); // 14.400px
-$font-smaller: type-scale(-2); // 11.520px
+$font-smaller: type-scale(-1); // DEPRECATED: Use $font-small.
```

Comment:
could you just set this to be `$font-smaller: $font-small` so if we make updates to $font-small you don't need to update it two places?